core/frontend/src/Viewport.ts

```diff
@@ -3572,6 +3572,7 @@ export class ScreenViewport extends Viewport {
     if (resized) {
       this.target.onResized();
       this.invalidateController();
+      this.target.forceBufferChange();
```

Comment:
Can you do this without having to make Viewport reach into RenderTarget's internals?

Reply:
> Can you do this without having to make Viewport reach into RenderTarget's internals?

I did not see a way to do that since the sizes have not changed and rendersToScreen is the only one who knows that webGLCanvas was resized (and that info is lost when it exits).  This does not happen in the normal resize view cases, just from rendersToScreen.

Reply:
Target.setRenderToScreen calls a function on SceneCompositor that sets SceneCompositor._width and _height to -1?

Reply:
Okay, that seems safe, but it is before updateViewRect and determining that it resized.  We only need it for the toScreen case, and I guess we can assume that it is highly likely that it will change size anyway?
Or did you want to just pass a true/false flag to onResized?
Also, in either case, do you prefer the SceneCompositor.forceBufferChange function I had added to set _width and _height rather than a setting a separate flag?

Reply:
> I guess we can assume

Yeah.

> do you prefer the SceneCompositor.forceBufferChange function I had added to set _width and _height rather than a setting a separate flag?

Yes, I prefer to avoid introducing a new flag.

Reply:
Done